libvast/test/flatbuffers.cpp

```diff
@@ -116,12 +119,13 @@ TEST(empty partition roundtrip) {
   // Prepare a mini meta index. The meta index only looks at the layout of the
   // table slices it gets, so we feed it with an empty table slice.
   auto meta_idx = vast::meta_index{};
-  vast::table_slice_header header;
-  header.layout = vast::record_type{{"x", vast::count_type{}}}.name("y");
-  header.offset = 0u;
-  header.rows = 0u;
-  auto slice = vast::msgpack_table_slice::make(header);
-  REQUIRE(slice);
+  auto layout = vast::record_type{{"x", vast::count_type{}}}.name("y");
+  auto slice_builder = vast::factory<vast::table_slice_builder>::make(
+    vast::defaults::import::table_slice_type, layout);
```

Comment:
Why did you switch away from the explicit `msgpack` format?

Reply:
We had an earlier effort to use the default table slice type in unit tests where possible, and this unit test was written at the same time as that effort, so it was overlooked.

Reply:
I see.